source/uwp/winui3/Visualizer/AdaptiveCardVisualizer.csproj

```
@@ -3,6 +3,7 @@
     <OutputType>WinExe</OutputType>
     <TargetFramework>net6.0-windows10.0.22000.0</TargetFramework>
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
+    <WindowsSdkPackageVersion>10.0.18362.38</WindowsSdkPackageVersion>
```

```suggestion
    <WindowsSdkPackageVersion>10.0.22000.38</WindowsSdkPackageVersion>
```

The TFM here is 22000, so it would use that respective SDK package version by default.

---

Whoops, I was misunderstanding what "TFM" stood for.